### PR TITLE
add style analyzer configuration

### PR DESCRIPTION
**Pull request by rporres**
Signed-off-by: Rafael Porres Molina <rafa@sourced.tech>

### CODE REVIEW

**Comment by bzz** [comment]
Enabled it on `bblfsh/uast-viewer`

File: .helm-staging.yml [hunk 1]
```diff
@@ -17,3 +17,10 @@ nodeSelector:
 providers:
   github:
     comment_footer: "_If you have feedback about this comment, please, [tell us](%s)._"
+
+analyzers:
+  - name: style-analyzer
+    addr: ipv4://lookout-style-analyzer:2000
+
+repositories:
```
Comment (rporres) [inline .helm-staging.yml]: This setting is not going to have any effect as this key is not taken into account. What are you trying to achieve here?

Comment (bzz) [reply .helm-staging.yml]: Thank you for notice! That is a way of adding more repositories for lookout to watch ([example](https://github.com/src-d/lookout/blob/master/config.yml.tpl))

Comment (rporres) [reply .helm-staging.yml]: Well, the chart does not currently support that... I will add it to the chart.

Comment (carlosms) [reply .helm-staging.yml]: Actually for some reason the repositories are defined with the positional argument, and this section of the config file is only used to optionally provide the credentials for those repos.
[Looking at the chart](https://github.com/src-d/charts/blob/master/lookout/templates/lookout-deployment.yaml#L38) I believe should be done with `repository` (already up in this file)

```
app:
  lookout:
    repository: github.com/src-d/lookout,github.com/bblfsh/uast-viewer
```

File: .helm-staging.yml [hunk 2]
```diff
@@ -17,3 +17,7 @@ nodeSelector:
 providers:
   github:
     comment_footer: "_If you have feedback about this comment, please, [tell us](%s)._"
+
+analyzers:
```
Comment (rporres) [inline .helm-staging.yml]: should we remove this? Chart is going to apply this configuration now

**Pull request merged**